README.md

```diff
@@ -1,2 +1,48 @@
-# playground-for-dart
+# Playground for Dart
+
 Simple examples that help you get started with Appwrite + Dart (=❤️)
```

Comment:
I would add a note about this being the playground for the server integration and add a link to the Flutter playground or SDK if that what someone looked for.